coop_membership/data/function.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<odoo>
+    <data noupdate="0">
+        <function name="_compute_registration_qty" model="shift.template">
+            <function name="search" model="shift.template">
+                <value eval="[]"/>
+            </function>
+        </function>
+        <function model="post.object"
+                  name="adjust_user_rights"/>
+    </data>
```

Comment:
@ivantodorovich  : I have remove it because it is just updating access rights for the static user

Here is the method in v9 : https://github.com/druidoo/FoodCoops/blob/9.0/intercoop_addons/coop_membership/data/post_object.py